src/libstd/sys/unix/ext.rs

```diff
@@ -121,20 +128,60 @@ impl OsStringExt for OsString {
     }
 }
 
-// Unix-specific extensions to `OsStr`.
+/// Unix-specific extensions to `OsStr`.
 pub trait OsStrExt {
+    /// Get the underlying byte view of the `OsStr` slice.
     fn as_byte_slice(&self) -> &[u8];
+
+    /// Convert the `OsStr` slice into a `CString`.
+    fn to_cstring(&self) -> CString;
```

Comment:
See http://aturon.github.io/style/naming/README.html -- we have a special case for single-letter "words".